Assets/Scripts/Network/RoomCreator.cs

```diff
@@ -1,6 +1,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
```

Comment:
Let's use the `TextMeshProUGUI` component instead of the Unity `Text` component for this message box, since the rest of the UI is using that already.